source/core/parameters_multiphysics.cc

```diff
@@ -453,6 +467,19 @@ Parameters::VOF_SurfaceTensionForce::parse_parameters(ParameterHandler &prm)
       verbosity = Parameters::Verbosity::quiet;
     else
       throw(std::runtime_error("Invalid verbosity level"));
+
+    prm.enter_subsection("marangoni effect");
+    {
+      enable_marangoni_effect = prm.get_bool("enable");
+
+      if (enable_marangoni_effect & !enable)
```

Comment:
I'd rename the enable variable to enable_STF or similar to make it clearer

Reply:
It was enable_STF, but me and Jeanne decided to change it to enable. I will discuss it with you.